commands/src/main/java/org/wildfly/extras/creaper/commands/socketbindings/RemoveSocketBinding.java

```diff
@@ -0,0 +1,48 @@
+package org.wildfly.extras.creaper.commands.socketbindings;
+
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+/**
+ * <p>
+ * A command for removing socket binding from default socket binding group if not specified else.</p>
+ * <p>
+ */
+public final class RemoveSocketBinding implements OnlineCommand {
+
+    private final String socketBindingName;
+    private String socketBindingGroup;
+
+    public RemoveSocketBinding(String socketBindingName) {
+        this(socketBindingName, null);
+    }
+
+    public RemoveSocketBinding(String socketBindingName, String socketBindingGroup) {
+        if (socketBindingName == null) {
+            throw new IllegalArgumentException("Name of the socket-binding must be specified as non null value");
+        }
+        if (socketBindingName.isEmpty()) {
+            throw new IllegalArgumentException("Name of the socket-binding must not be empty value");
+        }
+
+        this.socketBindingName = socketBindingName;
+        this.socketBindingGroup = socketBindingGroup;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Operations ops = new Operations(ctx.client);
+
+        if (socketBindingGroup == null) {
+            socketBindingGroup = ctx.client.options().isDomain ? "full-sockets" : "standard-sockets";
+        }
+
+        Address socketBindingAddress = Address.of("socket-binding-group", socketBindingGroup)
+                .and("socket-binding", socketBindingName);
+
+        ops.removeIfExists(socketBindingAddress);
```

Comment:
If the intended behaviour is to remove it lazily => only if it exists, the whole command should be either documented as such or have it configurable. As from `RemoveSocketBinding` by default I would expect that it would fail if such resource doesn't exist.

PS: I personally like the behaviour of having it removed lazily still it is inconsistent with other remove commands which fail if resource doesn't exist which is unfriendly from UX